Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "rate_extractor"
+version = "0.1.0"
+authors = ["name <harish.m@surya-soft.com>"]
```

Comment:
set your git global username, google it how u can do in git through command line